### PR TITLE
Update en.md gender language

### PR DESCRIPTION
**Pull request by tangerineshark**
All of this is a suggestion, not a required shift, so feel free to ignore it. I thought there might be a way to make the language a touch more gender-expansive, but also I tend to be verbose, so whether it's actually an improvement is your call.

### CODE REVIEW

**Comment by joostdecock** [comment]
Thank you for your input on this. I very much like the change from `gender-neutral` to `gender-inclusive`, great stuff.  
After that, I feel it's getting a bit wordy without doing much to clear up the possible confusion people might have.

This is not about patterns, it's about bodies. And one aimed at those who find this terminology confusing.
What I mean is, the people who reach out to us to say *I really appreciate the gender-inclusive phrasing on the website* don't need this explained to them. 

I tend to picture somebody like my mom when writing this stuff. Who is a sweet lady, but not exactly what you'd call *woke*. She is likely to be confused by this terminology, and I feel we need to explain this in a way that makes it intuitive to those who typically don't even think about this.

What do we mean by `with breasts` and `without breasts`? I feel the current incarnation does little to clear that up.

So I was hoping we could perhaps find something that is not as blunt as what I wrote, but still gets the message across in a way that makes sense to the kind of people who assume gender is binary.

Then again, this is certainly not the hill I'm going to die on :wink:

**Comment by l3xander** [comment]
I was actually thinking of making this suggestion as well, but I wasn't sure, I'm happy you did!

In regards to what Joost commented, "What do we mean by with breasts and without breasts? I feel the current incarnation does little to clear that up.", I suggest an explanation that goes like this:

_Nearly every clothing company has two main categories for their clothing: menswear and womenswear. These garment designs are usually based off of the ideal or average body. Meaning, for womenswear, the designer will likely make a garment for a body with breasts, narrow shoulders, a smaller waist and wide hips. These characteristics are generally thought of as "the typical characteristics of a woman's body". However, we believe it is best to stay away from this; no body is average and we want to build a gender-inclusive environment. We do not want to exclude anyone and we don't agree with the idea that "body shape = gender". That's why we use the terminology "with breasts" and "without breasts", simply asking whether a person has breast tissue or not._

(I should probably note that I'm not a professional, neither in writing nor in gender-inclusive topics (even though I am transgender).)

**Comment by joostdecock** [comment]
I've added the FAQ as we agreed on the call: https://next.freesewing.org/docs/faq/breasts/

I threw it together a bit from the inputs of both @tangerineshark and @Leo-Luuk and I updated the guide to link to the FAQ. There's almost certainly room for improvement there.

I will close this PR, as the ground has shifted and merging this will get messy.

Feel free to submit a new PR with improvements to the faq/guide or if you're not sure how yo do that, feel free to comment here and I'll implement your suggestions.

Thanks guys 🤗
